my_game.py

```diff
@@ -175,7 +177,9 @@ def __int__(self, shot_list, **kwargs):
 
         kwargs['filename'] = "images/ufoBlue.png"
 
-        kwargs['scale'] = SPRITE_SCALING
+        scale_change = random.randint(UFO_SIZE_1, UFO_SIZE_2)
```

Comment:
This only works if the sizes ARE x and x+1 (Like 1 and 2, or 5 and 6). It should work for any size. Like 3 and 7.